apps/aecore/src/aec_tx_processor.erl

```diff
@@ -857,7 +857,7 @@ contract_create({OwnerPubkey, Amount, Deposit, GasLimit, GasPrice,
     {Account, S1}  = get_account(OwnerPubkey, S),
     assert_account_balance(Account, TotalAmount),
     assert_contract_create_version(ABIVersion, VMVersion, S),
-    assert_contract_byte_code(ABIVersion, SerializedCode, CallData),
+    assert_contract_byte_code(ABIVersion, SerializedCode, CallData, S),
```

Comment:
So this is check in contract create tx. How does this interacts with state channels? Does this mean that contracts with new format created in state channels can be called in force progress tx, and also when tx on chain is under Roma protocol?